independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/DotNames.java

```diff
@@ -81,6 +83,8 @@
     public static final DotName NAMED = create(Named.class);
     public static final DotName ACTIVATE_REQUEST_CONTEXT = create(ActivateRequestContext.class);
     public static final DotName TRANSACTION_PHASE = create(TransactionPhase.class);
+    public static final DotName INITIALIZED = create(Initialized.class);
+    public static final DotName APPLICATION_SCOPED = create(ApplicationScoped.class);
```

Comment:
You can use `BuiltinScope.APPLICATION.getName()` instead...

Reply:
Good point, I forgot it's there.